File: cpp/leetcode/520.cpp
class Solution {
public:
    bool detectCapitalUse(string word) {
        int cnts = 0, n = word.size();
        for (int i = 0; i < n; ++i) {
            if (word[i] <= 'Z') {
                ++cnts;
            }
        }
        return cnts == 0 || cnts == n || (cnts == 1 && word[0] <= 'Z');
    }
};

File: python/leetcode/520.py
class Solution(object):
    def detectCapitalUse(self, word):
        cnts, n = 0, len(word)
        for ch in word:
            if ch <= 'Z':
                cnts += 1
        return cnts == 0 or cnts == n or (cnts == 1 and word[0] <= 'Z')